modal/cli/run.py

```diff
@@ -368,8 +375,9 @@ def get_command(self, ctx, func_ref):
 @click.option("-d", "--detach", is_flag=True, help="Don't stop the app if the local process dies or disconnects.")
 @click.option("-i", "--interactive", is_flag=True, help="Run the app in interactive mode.")
 @click.option("-e", "--env", help=ENV_OPTION_HELP, default=None)
+@click.option("-m", is_flag=True, help="Use a Python module path instead of a file path")
```

Comment:
Should the verb here be something like "Interpret"? Since it's just going to use whatever you give it, but we're trying to parameterize _how_ it uses it.